Sources/GtkCodeGen/GtkCodeGen.swift

```diff
@@ -224,6 +226,14 @@ struct GtkCodeGen {
                 return false
             }
 
+            //can cause problems with gtk versions older than 4.20.0
```

Comment:
Put a space after the `//` and capitalise `Can` at the start of the sentence (to match the style of other comments)